Text_to_Translated_Text_Step_2.ipynb

```diff
@@ -0,0 +1,360 @@
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "w0dZrIteLS1K",
+        "outputId": "049de07f-fe48-47a0-af68-b291bc64f447"
+      },
+      "execution_count": 128,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (2.31.0)\n",
+            "Requirement already satisfied: uuid in /usr/local/lib/python3.10/dist-packages (1.30)\n",
+            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests) (3.3.0)\n",
+            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests) (2.10)\n",
+            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests) (2.0.7)\n",
+            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests) (2023.7.22)\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Import required modules\n",
+        "import requests\n",
+        "\n",
+        "start_time = time.time()\n",
+        "\n",
+        "# Define your Azure Cognitive Services Translator API credentials and configuration\n",
+        "key = \"ebe0efc466cd4c119e987d8fb0a7c748\"\n",
+        "endpoint = \"https://api.cognitive.microsofttranslator.com/\"\n",
+        "\n",
+        "# Define the Azure region\n",
+        "location = \"westus2\"\n",
+        "\n",
+        "# Define the API path for translation\n",
+        "path = '/translate'\n",
+        "constructed_url = endpoint + path\n",
+        "\n",
+        "# Define parameters\n",
+        "params = {\n",
+        "    'api-version': '3.0',\n",
+        "    'from': 'en',\n",
+        "    'to': ['es']\n",
+        "}\n",
+        "\n",
+        "# Define request headers\n",
+        "headers = {\n",
+        "    'Ocp-Apim-Subscription-Key': key,\n",
+        "    'Ocp-Apim-Subscription-Region': location,\n",
+        "    'Content-type': 'application/json',\n",
+        "    'X-ClientTraceId': str(uuid.uuid4())\n",
+        "}\n",
+        "\n",
+        "# Define the text to be translated\n",
+        "body = [{\n",
+        "    'text': \"Hello there! My name is Phil. I have been an online English tutor for almost three years now. I've been helping students from beginner to advanced level improve their English skills. I've tried teaching kids and adults. I used CPR. Look! Rewards! And many more! I would love to help you improve and make progress in learning. From phonics, grammar, pronunciation and up to conversational English. I've already earned my Tassel and Tia Felt certificate, so you're in good hands. Learn with me and let's make English a fun and easy language. See ya!\"\n",
```

Comment:
Can you move all this text into a single global variable and move that to the top of the code so that it will be easier to test for multiple different texts and will also be a bit more readable?

Reply:
Thanks for pointing it out. Added the changes